CMakeLists.txt

```diff
@@ -440,7 +440,8 @@ ${CMAKE_BINARY_DIR}/generated-src/${filename}.h")
 
     # tray icon
     if(${SUNSHINE_ENABLE_TRAY})
-        pkg_check_modules(APPINDICATOR appindicator3-0.1)
+        pkg_check_modules(APPINDICATOR ayatana-appindicator3-0.1)
+        pkg_check_modules(LIBNOTIFY libnotify)
```

Comment:
This is now in `.\cmake\compile_definitions\linux.cmake`

Reply:
Also probably need to install all the icons for Linux?

That's in `.\cmake\packaging\linux.cmake`. Look for

`install(FILES "${CMAKE_SOURCE_DIR}/sunshine.svg" DESTINATION "${CMAKE_INSTALL_PREFIX}/share/icons")`

Reply:
I've fixed the ayatana build with conditional pkgconfig for both appindicator and ayatana-appindicator
I've also added the icons in the correct folder